monkey/infection_monkey/model/__init__.py

```diff
@@ -25,6 +25,7 @@
     CMD_PREFIX,
     MONKEY_ARG,
 )
+AGENT_OTP_ENVIRONMENT_VARIABLE = "INFECTION_MONKEY_AGENT_OTP"
```

Comment:
Lets move it to `monkey/infection_monkey/monkey.py`. This file should not exist.

Reply:
Also, can we make it something shorter like `IM_OTP`? We should also clean it up once we're done using it

Reply:
79f980b5363a14272cba272e931ac8ad8d44296a